File: Service/Implement/DanhMucCompanyTrangThaiService.cs
namespace Service.Implement
{
    public class DanhMucCompanyTrangThaiService : BaseService<DanhMucCompanyTrangThai, IDanhMucCompanyTrangThaiRepository>
    , IDanhMucCompanyTrangThaiService
    {
    private readonly IDanhMucCompanyTrangThaiRepository _DanhMucCompanyTrangThaiRepository;
    public DanhMucCompanyTrangThaiService(IDanhMucCompanyTrangThaiRepository DanhMucCompanyTrangThaiRepository) : base(DanhMucCompanyTrangThaiRepository)
    {
    _DanhMucCompanyTrangThaiRepository = DanhMucCompanyTrangThaiRepository;
    }
    }
    }

File: CMS/src/app/shared/DanhMucCompanyTrangThai.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { DanhMucCompanyTrangThai } from 'src/app/shared/DanhMucCompanyTrangThai.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class DanhMucCompanyTrangThaiService extends BaseService{
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "DanhMucCompanyTrangThai";
    }
}

